apps/site/pages/docs/api-extensions/extending-api-schema.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title: '  Extending API schemas'
+sidebar_label: 'API Extensions'
+---
+
+import { Callout, Step, Steps, FileTree, Tabs, Tab } from 'nextra-theme-docs'
+import { ExtendableQueryTable } from './../../../components/ExtendableQueryTable'
+
+<header>
+
+# Extending API schemas 
+
+</header>
+
+In this guide, learn how to extend VTEX and third-party APIs to customize them according to your store's needs, allowing you to efficiently retrieve and manipulate the necessary data.
+
+---
+
+## Before you start
+
+<Steps>
+### Avoid over-fetching data on pages
+ 
+Even though you can add information to the FastStore API schema, you must be careful not to over-fetch data on your pages. See the [best practices for fetching data on your storefront](/docs/api-extensions#best-practices-for-fetching-data).
+ 
+### Ensure your development server is active
+ 
+Ensure that your development server is up and running while developing to see the changes you make. You can do this by running the command `yarn dev` in your terminal, which will monitor the changes.
+
+</Steps>
+
+---
+
+## Extending VTEX API Schemas 
+
+FastStore streamlines the way of using data from VTEX APIs that the FastStore API does not expose.
+
+As FastStore uses GraphQL, it's required to write [type definitions](https://graphql.org/learn/schema/#object-types-and-fields) and [resolvers](https://graphql.org/learn/execution/#root-fields-resolvers) to fetch the data you need. 
+To define new type definitions and resolvers for FastStore using GraphQL, learn how to create a specific directory structure in the following step by step. 
+
+---
+
+### Step by step
+#### Step 1 - Preparing the folders and files 
+
+To declare a new type definition and resolver you need to create a directory structure:
+
+1. In you store repository, go to the `src` folder and create the `graphql` subfolder. You can do this by running the following command in your terminal:
+
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir graphql
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir graphql
+  ```
+  </Tab>
+</Tabs>
+
+2. Inside the new `graphql` folder, create the `vtex` subfolder.
+
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir vtex
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir vtex
+  ```
+  </Tab>
+</Tabs>
+
+3. In `vtex`, create `resolvers` and `typeDefs` subfolders.
+<Tabs items={['macOs and Linux', 'Windows']}>
+  <Tab>
+  ```bash copy
+  mkdir resolvers/typeDefs
+  ```
+  </Tab>
+
+  <Tab>
+   ```bash copy
+  mkdir resolvers\typeDefs
+  ```
+  </Tab>
+</Tabs>
+
+<Callout type="warning" emoji="⚠️">
+  The `src/graphql/vtex/resolvers` and `src/graphql/vtex/typeDefs` path is mandatory, but you can organize the files inside those directory as you wish.
```

Comment:
```suggestion
  The `src/graphql/vtex/resolvers` and `src/graphql/vtex/typeDefs` path are mandatory, but you can organize the files inside those directories as you wish.
```

Are they? You can have a store without these paths. In what sense are they mandatory? I also think you can have `src/graphql/vtex/resolvers` and don't have `src/graphql/vtex/typeDefs`, but @eduardoformiga can confirm this.